inputs/orchestrator_inner:1.json

```diff
@@ -35,6 +35,13 @@
   "prepublish_plugins": [],
   "exit_plugins": [
     {
+      "name": "store_metadata_in_osv3",
+      "args": {
+        "url": "{{OPENSHIFT_URI}}",
+        "verify_ssl": false
+      }
+    },
+    {
```

Comment:
This actually causes a test coverage decrease :-)
https://github.com/projectatomic/osbs-client/blob/3ee8276bfaafafe621d70dd9fb8a6a56498a0599/osbs/build/build_request.py#L273